### ISSUE TITLE
Update "add-prefix-in-playerlist" & "add-prefix-in-playerlist" default descriptions to make less confusing

### ISSUE THREAD

**Issue by axemwa** [EssentialsX/Essentials#2804]
### Prefixes and suffixes not appearing in tablist even if enabled

**Full output of `/ess version`**:

```
[02:44:09 INFO]: Server version: 1.13.2-R0.1-SNAPSHOT git-Paper-645 (MC: 1.13.2)
[02:44:09 INFO]: EssentialsX version: 2.17.1.15
[02:44:09 INFO]: LuckPerms version: 4.4.1
[02:44:09 INFO]: PlaceholderAPI version: 2.10.3
[02:44:09 INFO]: Vault version: 1.7.2-b107
[02:44:09 INFO]: EssentialsXGeoIP version: 2.17.1.15
```

**EssentialsX config**: [config.yml](https://pastebin.com/5SYe0ybm)

### Details

**Steps to reproduce**  

Enable Add-prefix-in-playerlist
Enable Add-suffix-in-playerlist. 
Start server
Check tablist in-game

**Expected behavior**  
Prefixes and Suffixes from permissions manager to appear in tablist

**Screenshots**  
![image](https://user-images.githubusercontent.com/37608633/66066383-959e4000-e588-11e9-8eb4-433e4069674c.png)

**Comment by mdcfe** [comment]
`add-prefix/suffix-in-playerlist` are both dependent on `add-prefix-suffix` being set to true - if they're set to false, the playerlist options won't have any effect.

The implementation of the various prefix and suffix config options is confusing - ideally they should be streamlined and documented to make more sense, because right now it's hard for anyone to figure out what combination of options does what.

**Comment by axemwa** [comment]
I agree, maybe in next update changing the default descriptions for those could be changed to something more understandable like so

![image](https://user-images.githubusercontent.com/37608633/66095844-37994900-e5d8-11e9-859f-2cf26573d6d6.png)

**Comment by JRoy** [comment]
Mostly addressed in #6108
